src/monodroid/CMakeLists.txt

```diff
@@ -116,6 +118,33 @@ set(TEST_COMPILER_ARGS_CPP ${XA_COMPILER_FLAGS}
   Wuninitialized
   )
 
+if(ENABLE_NDK)
+  if (ENABLE_CLANG_ASAN OR ENABLE_CLANG_UBSAN)
+	set(TEST_COMPILER_ARGS_CPP
```

Comment:
Please update `.editorconfig` so that `CMakeLists.txt` always expands tabs, then re-save this file so that tabs are spaces.